src/renderer/app-bridge.ts

```diff
@@ -200,13 +199,13 @@ export class AppBridge {
         );
         break;
       case apiCmds.notification:
-        notification.showNotification(
+        ssf.showNotification(
           data as INotificationData,
           this.callbackHandlers.onNotificationCallback,
         );
         break;
       case apiCmds.closeNotification:
-        await notification.hideNotification(data as number);
+        await ssf.closeNotification(data as number);
```

Comment:
For my own knowledge: what does ```ssf``` stand for?

Reply:
Symphony Software Foundation